### DISCUSSION TITLE
Defaulting to chrome's "custom tabs"

### DISCUSSION THREAD

**Discussion by radiumatic** [community/community#63436] [Mobile]
Opening links (in my case, release download link), forcefully, in chrome, really makes no sense.
In fact, in this matter I needed the link to be opened with my download manager. 
It would be better to allow the system defaults manager to decide the browser to be used

**Comment by alcere** [comment]
Hi @nimagp! I am sorry for the troubles. Please, let me know if I understood correctly the problem: Every time you open a link from the app, Chrome is used by default to download the file instead of using the web browser you have set as default from System settings?

**Reply by radiumatic** [reply]
> Hi @nimagp! I am sorry for the troubles. Please, let me know if I understood correctly the problem: Every time you open a link from the app, Chrome is used by default to download the file instead of using the web browser you have set as default from System settings? 

Hello, and sorry for the late reply. Not only downloads, but all links (even explicit github ones)
I believe it's because either github app or  android (or even MIUI) uses Chrome for in-app browser feature by default, with no choice to change it.

**Reply by alcere** [reply]
Hi @nimagp, thanks for reporting! We have recently made some improvements to how we handle external links. Have you noticed any improvement?

**Comment by Oddward** [comment]
> Opening links (in my case, release download link), forcefully, in chrome, really makes no sense.
> In fact, in this matter I needed the link to be opened with my download manager. 
> It would be better to allow the system defaults manager to decide the browser to be used

Not only that, but using the app doesn't improve your chances of successfully downloading a file at all. I've tried downloading the same app release 5 times now (logseq) on browser and mobile app, every time it fails somewhere after 50% and resuming a download never works as it becomes a "forbidden" direct link or something, I don't understand why.

Please implement some native downloading manager instead of opening a custom tab, it seems worse than using the mobile website, or have the mobile client be able to continue download sessions.

**Reply by alcere** [reply]
Hi @Oddward, I understand how relevant adding the option to download a file from the mobile app is and I have passed your feedback to our product team to evaluate how we can fit this into our backlog. Thank you very much for your understanding and patience.
